include/lo2s/line_info.hpp

```diff
@@ -32,33 +32,62 @@ namespace lo2s
 {
 struct LineInfo
 {
+private:
+    static const std::string unknown_file_str;
```

Comment:
I'd like to remove these ugly bastards and write the string literals directly at the few places where they are used.